src/main/resources/schemas/nhinc/common/PatientCorrelation.xsd

```diff
@@ -35,5 +38,12 @@
 		</xsd:sequence>
 	</xsd:complexType>
 	<xsd:element name="AddPatientCorrelationResponse" type="tns:AddPatientCorrelationResponseType"/>
+	<xsd:complexType name="AddPatientCorrelationPLQRequestType">
```

Comment:
I thought the plan was to use the existing Patient Correlation service and add the RLS ID as an optional element in the payload?